instrumentation/github.com/gin-gonic/gin/otelgin/test/gintrace_test.go

```diff
@@ -321,3 +328,61 @@ func TestWithGinFilter(t *testing.T) {
 		assert.Len(t, sr.Ended(), 1)
 	})
 }
+
+func TestSpanRecordError(t *testing.T) {
```

Comment:
Shouldn't this test be removed?

Reply:
Good idea, before this, I hadn't fully read the other tests. This is somewhat redundant.